AWSTranscribeStreaming/AWSTranscribeStreamingService.m

```diff
@@ -64,8 +66,8 @@ @interface AWSTranscribeStreaming()
 
 @property (nonatomic, strong) AWSNetworking *networking;
 @property (nonatomic, strong) AWSServiceConfiguration *configuration;
-@property (nonatomic, strong) AWSSRWebSocket *webSocket;
-@property (nonatomic, strong) AWSSRWebSocketDelegateAdaptor *webSocketDelegateAdaptor;
+@property (nonatomic, strong) id<AWSTranscribeStreamingWebSocketProvider> webSocketProvider;
+@property (nonatomic, strong) AWSSRWebSocketDelegateAdaptor *srWebSocketDelegateAdaptor;
```

Comment:
yep this was my comment on my own code too. im gonna make this change.

Reply:
okay I have to add callbackQueue as a prop if I do this to the delegate provider protocol. otherwise this code is no longer possible:
```
- (void)updateWebSocketDelegate {
    if (self.webSocketDelegateAdaptor) {
        [self.webSocketProvider setDelegateDispatchQueue:self.webSocketDelegateAdaptor.callbackQueue delegate: self.webSocketDelegateAdaptor];
        
    }
}
```

Reply:
after further discussion, not possible

Reply:
this should be fixed now